index.html

```diff
@@ -50,7 +50,6 @@
                 </label>
             </div>
             <div id="stats">
-                <p id="report"></p>
```

Comment:
It looks like you may have inadvertently deleted this `<p>`. You are still trying to grab it in your app.js and mutate the textContent, so I think it is needed.

Reply:
ah I had this problem as well at the very very end, and then I fixed it, so I'm guessing I forgot to make a new commit - will do and let you know